chef_master/source/server_backup_restore.rst

```diff
@@ -124,6 +127,76 @@ When restoring Chef server data, the previously backed-up files will be required
 
       $ chef-manage-ctl reconfigure
 
+
+Using Chef Backend
+=====================================================
+
+The backup and restore process allows you to restore a data backup
+into a newly built cluster in a disaster recovery scenario. It is not
+currently intended for the recovery of an individual machine in the
+chef-backend cluster or for point-in-time rollback of an existing
+cluster.
+
+Backup
+------
+
+To be able restore your data in the case of an emergency, you need
+backups of
+
+- the data in your Chef Backend cluster
+- the configuration from your Chef Server
+
+1. On a follower chef-backend node, run ``chef-backend-ctl backup``
+2. On a chef-server node run: ``chef-server-ctl backup --config-only``
+3. Move the tar archives created in steps (1) and (2) to a long-term
+   storage location.
+
+Restore
+-------
+
+To restore a Chef Backend-based Chef Server cluster:
+
+1. On the first machine that you want to use in your new Chef Backend
+   cluster. The argument to the ``--publish-address`` option should be
+   an IP address that can be used to reach the node you are restoring.
+
+.. code-block:: bash
+
+   chef-backend-ctl restore --publish-address X.Y.Z.W /path/to/backup.tar.gz
+
+
+2. Join additional nodes to your Chef Backend cluster (note: if you
+   are simply trying to test and verify your restore process, you can
+   skip the additional backend nodes)
+
+.. code-block:: bash
+
+   chef-backend-ctl join-cluster IP_OF_FIRST_NODE --publish-address IP_OF_THIS_NODE
+
+3. Restore a chef-server from your backed up chef-server configuration
+   (See step 2 in the Backup instructions). Alternatively, you can
```

Comment:
This one is a bit hard to link to because there are no anchors for the individual steps in a numbered list.  I can link to the backup instructions themselves though.